mmv1/products/networkservices/EdgeCacheService.yaml

```diff
@@ -191,7 +191,7 @@ properties:
                 The name of the pathMatcher associated with this hostRule.
               required: true
         min_size: 1
-        max_size: 10
+        max_size: 50
```

Comment:
These _look_ like they're in the wrong spot in the YAML config, but from the diff and my functional testing, I believe they're in the right spot. Let me know if there's a way to move or indent them so it's more obvious which attributes they line up with.